script/controller/tasklist-front.js

```diff
@@ -76,27 +100,35 @@ angular.module('tasklist-front', ['tasklist-back', 'users-back', 'utils'])
             }
         };
 
-        $scope.$watch('assignedByMe', function () {
-            if ($scope.assignedByMe) {
-                $state.transitionTo('tasks.assignedByMe');
-            } else {
-                $state.transitionTo('tasks.all');
+        $scope.onBlur = function (item) {
```

Comment:
we discussed that. try not to have methods in controller like "onBlur", this is UI-related stuff, please have naming that have meaning in terms of business model